### DISCUSSION TITLE
Is there a way to point to a Content Library for ISO files?

### DISCUSSION THREAD

**Discussion by akulbe** [vmware/packer-examples-for-vsphere#806] [Help] [answered]
I've taken to putting my ISO files in a Content library.

In common.pkrvars.hcl

There's this section:

// Removable Media Settings                                           
common_iso_datastore = "datastore1"

If I put the directory structure "iso/linux/ubuntu/<ubuntu ISO file>" in the common_iso_datastore

it works… but if it's possible, I'm wondering if there's a way to just point to <content_library>/installer.iso and have builds work?

Thanks!

**Comment by tenthirtyam** [comment, marked as answer]
Yes, it's totally possible by the plug-in but not included in the samples. Feel free to submit an enhancement request to allow this to be used.
